dataprofiler/profilers/profile_builder.py

```diff
@@ -1307,6 +1362,19 @@ def report(self, report_options: dict = None) -> dict:
         report["data_stats"] = self._profile.report(remove_disabled_flag)
         return _prepare_report(report, output_format, omit_keys)
 
+    @classmethod
```

Comment:
if we are just calling super, no need to include, super contains the work.

However, nothing lower level for unstruct is implemented so better to raise a not implemented error.